lib/vagrant-libvirt/action/handle_box_image.rb

```diff
@@ -64,9 +64,9 @@ def call(env)
           # locking all subsequent actions as well.
           @@lock.synchronize do
             # Don't continue if image already exists in storage pool.
-            break if ProviderLibvirt::Util::Collection.find_matching(
-              env[:machine].provider.driver.connection.volumes.all, env[:box_volume_name]
-            )
+            break if env[:machine].provider.driver.connection.volumes.all(
+              name: env[:box_volume_name]
+            ).first.id
```

Comment:
does the volumes.all always return something, otherwise I'd imagine this could result in a nil access error if the list is empty?